src/gui/screen_menu_lan_settings.hpp

```diff
@@ -12,7 +12,7 @@
 // Container for this base class contains all MI from both ETH and WIFI screen
 // There can be MI, that will not be used in derived class (MI_WIFI_... won't be used in ETH Screen)
 // This is a solution to the problem that base class container have to define what MIs will be used, but derived classes will have slightly different ones.
-using ScreenMenuConnectionBase__ = ScreenMenu<GuiDefaults::MenuFooter, MI_RETURN, MI_WIFI_STATUS_t, MI_WIFI_INIT_t, MI_WIFI_CREDENTIALS_INI_FILE_t, MI_WIFI_CREDENTIALS_t, MI_NET_IP_VER_t, MI_NET_IP_t, MI_IP4_ADDR, MI_IP4_NMSK, MI_IP4_GWAY, MI_MAC_ADDR>;
+using ScreenMenuConnectionBase__ = ScreenMenu<EFooter::Off, MI_RETURN, MI_WIFI_STATUS_t, MI_WIFI_INIT_t, MI_WIFI_CREDENTIALS_INI_FILE_t, MI_WIFI_CREDENTIALS_t, MI_NET_IP_VER_t, MI_NET_IP_t, MI_IP4_ADDR, MI_IP4_NMSK, MI_IP4_GWAY, MI_MAC_ADDR>;
```

Comment:
Seems like at some places, it changes from `GuiDefaults::MenuFooter` to something, some other places have it the other way around.

Can you explain why? Would it have been better to make the change in a separate PR to make it easier to review?

Reply:
You have it in separate commit.
GuiDefaults::MenuFooter is default setting (of for MK4 on for MINI)
EFooter::Off is of for all printers

It was wrong in public and GuiDefaults::MenuFooter was set to off for NINI too